internal/provider/schema_cloudflare_access_application.go

```diff
@@ -46,8 +46,8 @@ func resourceCloudflareAccessApplicationSchema() map[string]*schema.Schema {
 			Type:         schema.TypeString,
 			Optional:     true,
 			Default:      "self_hosted",
-			ValidateFunc: validation.StringInSlice([]string{"self_hosted", "saas", "ssh", "vnc", "bookmark"}, false),
-			Description:  fmt.Sprintf("The application type. %s", renderAvailableDocumentationValuesStringSlice([]string{"self_hosted", "saas", "ssh", "vnc", "bookmark"})),
+			ValidateFunc: validation.StringInSlice([]string{"app_launcher", "bookmark", "dash_sso", "saas", "self_hosted", "ssh", "vnc", "warp"}, false),
+			Description:  fmt.Sprintf("The application type. %s", renderAvailableDocumentationValuesStringSlice([]string{"app_launcher", "bookmark", "dash_sso", "saas", "self_hosted", "ssh", "vnc", "warp"})),
```

Comment:
missing BISO here too - https://api.cloudflare.com/#access-applications-add-the-clientless-web-isolation-application

```suggestion
			ValidateFunc: validation.StringInSlice([]string{"app_launcher", "biso", "bookmark", "dash_sso", "saas", "self_hosted", "ssh", "vnc", "warp"}, false),
			Description:  fmt.Sprintf("The application type. %s", renderAvailableDocumentationValuesStringSlice([]string{"app_launcher", "biso", "bookmark", "dash_sso", "saas", "self_hosted", "ssh", "vnc", "warp"})),
```